src/condor_schedd.V6/schedd.cpp

```diff
@@ -13034,11 +13045,16 @@ Scheduler::Init()
 			"WARNING: CONDOR_ADMIN not specified in config file" );
 	}
 
+
+	ignore_domain_mismatch_when_setting_owner = param_boolean("TRUST_UID_DOMAIN", false);
+	ignore_domain_for_OwnerCheck = param_boolean("IGNORE_DOMAIN_FOR_JOB_OWNER_CHECK", ignore_domain_for_OwnerCheck);
+	warn_domain_for_OwnerCheck = param_boolean("WARN_DOMAIN_FOR_JOB_OWNER_CHECK", warn_domain_for_OwnerCheck);;
+	job_owner_must_be_UidDomain = param_boolean("JOB_OWNER_MUST_BE_FROM_UID_DOMAIN", job_owner_must_be_UidDomain);
```

Comment:
This is a weird use of default values. The knobs' values are sticky during the lifetime of the schedd if the parameters disappear from the config files. Is that intentional?

Reply:
That's more of a side effect.  I was thinking mostly about using the compile time init value as the default value here.

Reply:
I'd put the explicit true/false values in the param_boolean() calls. The compile-time initialized values should be irrelevant, since the first call to Init() will overwrite them (before they're used). The stickiness of the the values may confuse people, with no benefit.

Reply:
commit added to PR